storage/object/api-cli/combining-iam-and-object-storage.mdx

```diff
@@ -1,280 +1,174 @@
-}
-```
-Whenever an action is not explicitly allowed, it is automatically denied. In the bucket policy below we explicitly deny the `GetObject` action, but do not allow any others. This means that by default all actions will be denied.
-```json
-{
-    "Version": "2023-04-17",
-    "Id": "production-bucket-1-policy",
-    "Statement": [
-        {
-            "Sid": "deny get object",
-            "Effect": "Deny",
-            "Principal": {
-                "SCW": "user_id:81a2ad27-2273-4bf9-976f-3f06957ab6e1"
-            },
-            "Action": [
-                "s3:GetObject"
-            ],
-            "Resource": [
-                "production-bucket-1/*"
-            ]
-        }
-    ]
-}
-```
-
-### Giving an application full access to a bucket
-
-In the example below, we provide the IAM and bucket policy configurations needed to give an application within an IAM group full access to a bucket. This means only one application within the IAM group will be able to perform any of the [available S3 actions](/storage/object/api-cli/bucket-policy/#supported-actions) on the bucket.
-
-An IAM policy must be created to grant full access to Object Storage resources in Project 1 to group Alpha, which includes application A, user B and application C.
-```json
-{
-   "name": "policy-S3-proj1",
-   "description": "This policy grants full access to Object Storage resources in Project 1 to applications A and C, and user B",
-   "organization_id": "946552dd-a7dc-4d35-81df-11532f025b28",
-   "rules": [
-     {
-      "permission_set_names": [
-         "ObjectStorageFullAccess"
-      ],
-      "project_ids": [
-         "c6842bac-7938-4c04-9e03-f48147eee1f1"
-      ]
-     }
-   ],
-   "group_id": "ec204230-2011-4ac6-b86e-1d960b4c54cd"
-}
-```
-In the bucket policy below, we declare that all available actions are allowed for application A. We set the action field to `*` to include all available bucket actions in the statement, and we set the effect to `Allow`.
-
-Even if, in this example, all users and applications in group Alpha have full access to Object Storage through the IAM policy, the existing bucket policy only explicitly allows actions for application A. This means all other members in the group are automatically denied permissions to perform actions, since they are not explicitly allowed to do so.
-```json
-{
-   "Version": "2023-04-17",
-    "Id": "production-bucket-1-policy",
-    "Statement": [
+### Setting up access
+
+1. [Create a group](/identity-and-access-management/iam/how-to/create-group/) named `S3-group` that contains the user and the application, so they are the only ones to have access to Object Storage.
+
+2. [Create an IAM policy](/identity-and-access-management/iam/how-to/create-policy/) named `S3-policy` with the previously created group as a [principal](identity-and-access-management/iam/concepts/#principal). The members of this group will have the `ObjectStorageFullAccess` right.
```

Comment:
We could say that this can be done easily in the console, but this is also a way to do it. Have you tried to push this policy with the API to make sure it works?

Reply:
removed the IAM policy json (it was in the old doc) as there is no documentation on how to write and push iam policies

Reply:
and added a link to console